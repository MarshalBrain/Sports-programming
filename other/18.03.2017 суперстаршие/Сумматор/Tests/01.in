1 10
2 1 8
3 8 5201
3 4 -4008
2 8 8
3 10 -6997
2 3 4
3 9 1134
3 10 747
2 2 7
2 5 10
