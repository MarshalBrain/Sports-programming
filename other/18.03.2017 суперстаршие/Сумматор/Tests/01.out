-8
-8
-16
-20
-20
